openwisp_utils/admin_theme/apps.py

```diff
@@ -25,5 +25,5 @@ def register_menu_groups(self):
         # register dashboard item
         register_menu_group(
             position=10,
-            config={'label': _('Home'), 'url': '/admin', 'icon': 'ow-dashboard-icon'},
+            config={'label': _('Home'), 'url': '/admin/', 'icon': 'ow-dashboard-icon'},
```

Comment:
I hope the code isn't flaky enough that the presence of the last dash matters and not having it would break it.
~If it matters, it might be in our best interest to change the django side of code to use `urljoin` instead of appending strings to make the code more robust! :smile:
What do you think?~

Reply:
Yes, that slash matters. 
I think we need to change URL for the custom link. We can add ``/`` in the end of url provided by user only if it is missing.